README.md

```diff
@@ -1,2 +1,19 @@
-# contracts
-fully automated luxury gay space communism contracts
+# circles-contracts
+
+This is the initial smart contract implementation for the Circles universal basic income platform.
+
+**Note:** This is not yet intended for deployment in a production system.
+
+## Basic design
+
+In general the design philosophy here was to favor restriction of outside interference in token state. The separation of individual token logics into discrete contracts ensures invariants are maintained for the stakeholders (e.g. Circles can never mess with your individual token balance or issuance)
```

Comment:
I don't understand why the use of the `CirclesPerson` contract provides these guarantees?

Reply:
This doesn't really have anything to do with the `CirclesPerson` pattern. In the `sketch` contracts, when a user joined a `circles registry` the registry had the authority to transfer your tokens (given some initial condition). The fact that this registry doesn't exist any more means that only your deployed contract can touch your tokens.